scripts/install.sh

```diff
@@ -116,7 +116,7 @@ function check_dist() {
 	dist=$(echo "${dist}" | tr '[:upper:]' '[:lower:]')
 
 	case "${dist}" in
-		debian|ubuntu|centos|fedora)
+		debian|ubuntu|centos|fedora|almalinux)
 			echo "OK"
 			;;
 		cloudlinux)
```

Comment:
Since almalinux is not officially supported by DO, can you move almalinux down to this block with cloudlinux ? \\

Reply:
Sure. Will submit another PR with that change.